### PR TITLE
Feature/pretty filter

### PR DESCRIPTION
**Pull request by klaseskilson**
Make the filter part look nicer, more separated from the rest of the file view ✨

skarmavbild 2015-06-02 kl 17 22 13

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<footer id="footer">
+<footer id="footer" class="container">
```
Comment (klaseskilson) [inline app/views/layouts/_footer.html.erb:1]: image

**Comment by danielronnkvist** [comment]
The upload form crashes since `@bruse_file` in the form is `nil`. :cry:

**Comment by tkomstadius** [comment]
https://www.google.se/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0CAcQjRw&url=http%3A%2F%2Fwww.quickmeme.com%2Fmeme%2F35713x&ei=pM5tVbiPMIb9ygPqiYPwCg&bvm=bv.94455598,d.bGQ&psig=AFQjCNGj84DZfLl3eu1dfow4eR_tw-Brxw&ust=1433346079082525

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (klaseskilson) [inline app/views/layouts/_footer.html.erb]: image

**Comment by tkomstadius** [comment]
9f2eeb0e221b0f8d1cf270ee7e6c6c18aab7716618984e61c7fcfae02e8ca55d

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (klaseskilson) [inline app/views/layouts/_footer.html.erb:1]: image

**Comment by danielronnkvist** [comment]
Everything works! Looks nice!

**Comment by tkomstadius** [comment]
Is pretty 
86b78e69ae97e5cde514757bec2c07cb

**Pull request merged**